api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1989,6 +1997,57 @@ public void removeBucket(String bucketName)
    * @param bucketName  Bucket name.
    * @param objectName  Object name to create in the bucket.
    * @param fileName    File name to upload.
+   * @param contentType File content type of the object, user supplied.
+   *
+   * @throws InvalidBucketNameException  upon invalid bucket name is given
+   * @throws NoResponseException         upon no response from server
+   * @throws IOException                 upon connection error
+   * @throws XmlPullParserException      upon parsing response xml
+   * @throws ErrorResponseException      upon unsuccessful execution
+   * @throws InternalException           upon internal library error
+   */
+  public void putObject(String bucketName, String objectName, String fileName, String contentType)
```

Comment:
Make `public void putObject(String bucketName, String objectName, String fileName)` method to use this version of method to avoid code duplication.